handler/setup.go

```diff
@@ -200,6 +202,22 @@ func getDockerSockVolume() *spec.Volume {
 	}
 }
 
+func appendConfigVolume(volumes []*spec.Volume) []*spec.Volume {
+	loadedConfig, _ := config.Load()
+	droneRunnerVolumes := loadedConfig.DRONE_RUNNER_VOLUMES
+	for _, droneRunnerVolume := range droneRunnerVolumes {
+		volume := &spec.Volume{
+			HostPath: &spec.VolumeHostPath{
+				Name: strings.Split(droneRunnerVolume, ":")[0],
+				Path: strings.Split(droneRunnerVolume, ":")[0],
+				ID:   random(),
```

Comment:
Make sure this is able to handle the case where we want the same local file mounted on different paths in the containers - eg:
`/path/to/a.txt:/path/to/b.txt,/path/to/a.txt:/container/path/to/a.txt,...`
You can have a look as to how it's done here if the above case is not working -  https://github.com/drone-runners/drone-runner-aws/pull/88/files - this should handle all the cases

Reply:
yeah made sure of it